src/Opdex.Platform.WebApi/Opdex.Platform.WebApi.csproj

```diff
@@ -44,6 +43,22 @@
       <ProjectReference Include="..\Opdex.Platform.Infrastructure\Opdex.Platform.Infrastructure.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="Attributes\**" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <EmbeddedResource Remove="Attributes\**" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <None Remove="Attributes\**" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Content Remove="Attributes\**" />
+    </ItemGroup>
+
```

Comment:
Whats going on here, is it necessary?

Reply:
Maybe we have a folder or something named `Attributes` that needs to be deleted?

Reply:
Yeah, I've removed it and will commit with other changes